pkg/otlp/config.go

```diff
@@ -85,23 +86,29 @@ func FromAgentConfig(cfg config.Config) (PipelineConfig, error) {
 	debugConfig := readConfigSection(cfg, config.OTLPDebug)
 
 	return PipelineConfig{
-		OTLPReceiverConfig: otlpConfig.ToStringMap(),
-		TracePort:          tracePort,
-		MetricsEnabled:     metricsEnabled,
-		TracesEnabled:      tracesEnabled,
-		Metrics:            metricsConfig.ToStringMap(),
-		Debug:              debugConfig.ToStringMap(),
+		OTLPReceiverConfig:       otlpConfig.ToStringMap(),
+		OpenCensusReceiverConfig: censusConfig.ToStringMap(),
+		OpenCensusEnabled:        hasSection(cfg, "opencensus"),
+		TracePort:                tracePort,
+		MetricsEnabled:           metricsEnabled,
+		TracesEnabled:            tracesEnabled,
+		Metrics:                  metricsConfig.ToStringMap(),
+		Debug:                    debugConfig.ToStringMap(),
 	}, multierr.Combine(errs...)
 }
 
 // IsEnabled checks if OTLP pipeline is enabled in a given config.
 func IsEnabled(cfg config.Config) bool {
```

Comment:
The way things are built right now, you can only use the OpenCensus receiver if you also enable the OTLP receiver. Is this intentional?

Reply:
Hmm, it was intentional, yes. Do you think it should be otherwise? That would make things quite complicated in the code and I'm not sure it's worth the effort.

Reply:
I updated the PR description to reflect that.

Reply:
I think it's fine to leave it like this if it's internal